lib/src/main/java/org/team100/lib/motor/turning/Falcon6TurningMotor.java

```diff
@@ -63,7 +63,7 @@ public class Falcon6TurningMotor implements Motor100<Angle100> {
      * Proportional feedback coefficient for the controller. The error is measured
      * in sensor units (ticks per 100ms), and the full scale output is 1023.
      */
-    private static final double outboardP = 0.1;
+    private static final double outboardP = 1/2048;
```

Comment:
change the comment to match the meaning you have in your head when you write this number.  error measured in ... rev/s?  also don't use an expression like this, just write a literal.